packages/swirl-components/src/components/swirl-tree-view-item/swirl-tree-view-item.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Fragment,
+  h,
+  Host,
+  Method,
+  Prop,
+  State,
+} from "@stencil/core";
+import classNames from "classnames";
+
+@Component({
+  scoped: true,
+  shadow: false,
+  styleUrl: "swirl-tree-view-item.css",
+  tag: "swirl-tree-view-item",
+})
+export class SwirlTreeViewItem {
```

Comment:
**todo:** Missing Slot decorator

Reply:
Done.

Reply:
There's still the slot missing in `swirl-tree-view-item` for the `tags`

Reply:
Oops. Done.